src/internal/m365/service/onedrive/mock/handlers.go

```diff
@@ -316,7 +316,7 @@ func (edibd EnumerateItemsDeltaByDrive) EnumerateDriveItemsDelta(
 	_ api.CallConfig,
 ) api.NextPageResulter[models.DriveItemable] {
 	didp := edibd.DrivePagers[driveID]
-	return &didp
+	return didp
```

Comment:
this probably does need to return the pointer to the struct. Otherwise `NextPage` probably won't work as expected (which is why I'm guessing the diff for the index increment)

Reply:
That is a pointer.  See line 301: `DrivePagers map[string]*DriveItemsDeltaPager`